### ISSUE TITLE
Truncated timemory jsons

### ISSUE THREAD

**Issue by glpuga** [Ekumen-OS/lambkin#111]
### Bug description

Running 24hr bagfiles with timememory profiling on ends up with broken records because the recorded timememory output jsons get truncated and become unloadable.

Manually fixing them by emtying (not removing) the "history" section makes them usable again.

An untested solution may be to increase the sigterm and sigkill timeouts in ros2 launch

**Platform (please complete the following information):**
 - OS: [e.g. Ubuntu Focal]
 - Python version: [e.g. 3.8]
 - `lambkin` version: [e.g. tag, commit sha]

### How to reproduce
_List steps to reproduce the issue:_
1. ...

_Code snippets or minimal examples are always helpful, if not necessary._

**Expected behavior**
_A clear and concise description of what you expected to happen._

**Actual behavior**
_A clear and concise description of what you actually happened._

### Additional context
Any other information you think could be meaningful to this issue.

**Comment by hidmic** [comment]
> An untested solution may be to increase the sigterm and sigkill timeouts in ros2 launch

Yeah. It's brittle but it's an option. The actual solution would for the profiler to do incremental writes to storage. Considering https://github.com/NERSC/timemory has been archived, perhaps there are other profiling tools we can use.

**Comment by glpuga** [comment]
The information we want from the json is actually volunteered by timememory in the on the standard output, can we get it form there instead?
